catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrCatalogProvider.java

```diff
@@ -114,7 +115,8 @@
      * near real time index yet.  Cache is checked when ID queries fail from Solr.
      */
     private Cache<String, Metacard> pendingNrtIndex = CacheBuilder.newBuilder()
-            .expireAfterWrite(5, TimeUnit.SECONDS)
+            .expireAfterWrite(NumberUtils.toInt(System.getProperty("solr.provider.pending-nrt-ttl"),
```

Comment:
How hidden should this hidden attribute be? Will we document it anywhere? Should it be a constant?